tests/integration/component.go

```diff
@@ -712,4 +713,72 @@ func componentTests(args ...string) {
 			Expect(stdError).To(ContainSubstring("quantities must match the regular expression"))
 		})
 	})
+
+	Context("Creating component using symlink", func() {
+		JustBeforeEach(func() {
+			project = helper.CreateRandProject()
+			originalDir = helper.Getwd()
+			helper.Chdir(filepath.Dir(context))
+		})
+		JustAfterEach(func() {
+			// remove the symlinks
+			for _, symLinkPath := range symLinkPaths {
+				err := os.Remove(symLinkPath)
+				Expect(err).NotTo(HaveOccurred())
+			}
+
+			helper.DeleteProject(project)
+			helper.Chdir(originalDir)
+		})
+
+		It("Should be able to deploy a spring boot uberjar file using symlinks in all odo commands", func() {
+			oc.ImportJavaIS(project)
+
+			// create a symlink
+			symLinkName := helper.RandString(10)
+			helper.SymLink(filepath.Base(context), symLinkName)
+			symLinkPath := filepath.Join(filepath.Dir(context), symLinkName)
+			symLinkPaths = append(symLinkPaths, symLinkPath)
+
+			helper.CopyExample(filepath.Join("binary", "java", "openjdk"), context)
+
+			// create the component using symlink
+			helper.CmdShouldPass("odo", "create", "java:8", "sb-jar-test", "--project",
```

Comment:
Can you please follow the format like
```
helper.CmdShouldPass("odo", append(args, "create", "java:8", "sb-jar-test", "--project", project, "--binary", filepath.Join(symLinkPath, "sb.jar"), "--context", symLinkPath)...)
```
and replace the same format in remaining occurrence